### ISSUE TITLE
Error in LSPCompletionContributor when completing git messages in commit dialog

### ISSUE THREAD

**Issue by gayanper** [ballerina-platform/lsp4intellij#48]
Triggering completion git dialog throws the following error
```
java.lang.NullPointerException
	at java.util.concurrent.ConcurrentHashMap.get(ConcurrentHashMap.java:936)
	at org.wso2.lsp4intellij.editor.EditorEventManagerBase.forUri(EditorEventManagerBase.java:93)
	at org.wso2.lsp4intellij.contributors.LSPCompletionContributor.invokeAutoPopup(LSPCompletionContributor.java:56)
	at com.intellij.codeInsight.editorActions.TypedHandler.isAutoPopup(TypedHandler.java:290)
	at com.intellij.codeInsight.editorActions.TypedHandler.autoPopupCompletion(TypedHandler.java:273)
	at com.intellij.codeInsight.editorActions.TypedHandler.lambda$execute$0(TypedHandler.java:180)
	at com.intellij.openapi.editor.impl.CaretModelImpl.lambda$runForEachCaret$3(CaretModelImpl.java:354)
	at com.intellij.openapi.editor.impl.CaretModelImpl.doWithCaretMerging(CaretModelImpl.java:453)
	at com.intellij.openapi.editor.impl.CaretModelImpl.runForEachCaret(CaretModelImpl.java:346)
```
Seems like the LSPCompletionContributor need some guarding.

**Comment by NipunaRanasinghe** [comment]
@gayanper Thanks for reporting! :) is this intermittent or if not, could you provide a way to reproduce the issue??

**Comment by gayanper** [comment]
*Steps to reproduce*
- Install the lsp client plugin, in my case it was the WIP sts4 plugin
- without triggering the lang server to startup by opening any files, open git commit dialog. 
- press ctrl+space to trigger auto completion

I will be able to provide a fix tonight 🙂

**Comment by NipunaRanasinghe** [comment]
Closing since fixed by #73. Credit goes to @gayanper once more :)
